lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -469,14 +472,28 @@ void OpenCLFunction::executeConvolution(const OCLConvolutionInst *CC) {
 
   // Generate a tailor-made convolution kernel using the provided options based
   // on the parameters of the current convolution.
-  auto prog = createProgram(FWD_CONV_CODE, options, commands_);
-  auto kernel = createKernel("conv_forward_mem", prog);
+  auto &programName = isQuantized ? FWD_CONV_QUANTIZED_CODE : FWD_CONV_CODE;
+  auto prog = createProgram(programName, options, commands_);
+  auto kernelName = isQuantized ? "conv_forward_mem_i8" : "conv_forward_mem";
+  auto kernel = createKernel(kernelName, prog);
   setKernelArg(kernel, 0, deviceBuffer_);
   setKernelArg<cl_uint>(kernel, 1, tensors_[input]);
   setKernelArg<cl_uint>(kernel, 2, tensors_[weights]);
   setKernelArg<cl_uint>(kernel, 3, tensors_[bias]);
   setKernelArg<cl_uint>(kernel, 4, tensors_[output]);
 
+  // Extra options for quantized kernel
+  if (isQuantized) {
+    setKernelArg(kernel, 5, CC->getFilter()->getType()->getOffset());
```

Comment:
At the beginning of the function we have the following definitions:
```c++
 auto input = CC->getSrc();
  auto output = CC->getDest();
  auto bias = CC->getBias();
  auto weights = CC->getFilter();
```
Please use them here to shorten the code, instead of writing `CC->get...`